urllib3/connectionpool.py

```diff
@@ -652,7 +652,15 @@ def urlopen(self, method, url, body=None, headers=None, retries=None,
 
         # Check if we should retry the HTTP response.
         if retries.is_forced_retry(method, status_code=response.status):
-            retries = retries.increment(method, url, response=response, _pool=self)
+            try:
+                retries = retries.increment(method, url, response=response, _pool=self)
```

Comment:
I don't see where that extra time is. I merely wrapped increment with try...except